### ISSUE TITLE
Native crash with SnapshotSelectUIResponse_GetData

### ISSUE THREAD

**Issue by rmforti** [playgameservices/play-games-plugin-for-unity#2640]
Hello,

We've been developing an app for android and we are getting some random crashes on libgpg.so. After simbolicating we were able to track the issue down to the function **SnapshotManager_SnapshotSelectUIResponse_GetData.**

The weird part is that we do not use Native UI for SaveGame handling in our app whatsoever. Therefore, there are no function calls to `ShowSelectSavedGameUI`, which is the only access to C# wrapper function that could possibly call this method. (I've also checked all script defined functions)

Environment:
Unity 2018.2.6f1
Plugin Version: gpgs-plugin-support-0.9.58
Android Version: 8.0 and 9

**Error example:**
```
java.lang.Error: signal 11 (SIGSEGV), code 2 (SEGV_ACCERR), fault addr d27ffbf4
Build fingerprint: 'HUAWEI/FIG-LX1/HWFIG-H:9/HUAWEIFIG-L31/115C10R1:user/release-keys'
Revision: '0'
pid: 18605, tid: 18843, name: callback_queue  >>> com.noodlecake.farmpunks <<<
r0 c624afc0  r1 c72e5438  r2 d27ffbf4  r3 c62a2564
r4 c1ec68a8  r5 c767dfd8  r6 c624afc0  r7 c2e776e0
r8 c2e776f0  r9 00000001  sl c1ec68f8  fp c4cdf138
ip c2e7213c  sp c1ec68a8  lr c2dbf3d3  pc d27ffbf4  cpsr eca38c90
at Unknown.00102bf4(Unknown Source
  at libgpg.001a83d1 (Native Method)
  at libgpg.001823ed (Native Method)
  at libc.__pthread_start
  at libc.__start_thread (__start_thread:32)
```

**Comment by rmforti** [comment]
Still stuck on this!

**Comment by olehkuznetsov** [comment]
Should be fixed in v0.10.06. Feel free to reopen if it still crashes there.
